### PR TITLE
[Issues 6606] Add updated slack-guidelines for moderators

### LINKED ISSUE #6606: [TO DO] Add words to the slack guidelines for new slack channel creation
### Describe the issue

This more centers around slack administrators and the PR review & approval process

/assign markyjackson-taulia 

/sig contributor-experience

### PR DESCRIPTION
**Pull request by markyjackson-taulia**
**Which issue(s) this PR fixes**:

Fixes #6606

### CODE REVIEW

**Review by coderanger — COMMENTED** [review]
/lgtm

File: communication/slack-guidelines.md [hunk 1]
```diff
@@ -162,8 +162,10 @@ To add a channel, open a Pull Request (PR) updating the [slack-config][].
     -   Special accommodations will be made where necessary.
 
 After you submit your request the Slack Admins will review and follow-up with
-any questions in the PR itself. Once it is signed off and merged, the channel
-will be created.
+any questions in the PR itself. 
+There are two approvals needed. `/lgtm` and `/approved`. Once one moderator give the `/lgtm`, a hold will be placed on the PR using `/hold`. This hold will remain in place until one or more moderators reviews 
+and add the `/approve` label as well as `/hold cancel`, which will remove the hold on the PR.
```
Comment (cblecker) [inline communication/slack-guidelines.md:167]:
```suggestion
There are two approvals needed. `/lgtm` and `/approve`. Once one moderator give the `/lgtm`, a hold will be placed on the PR using `/hold`. This hold will remain in place until one or more moderators reviews 
and add the `/approve` command as well as `/hold cancel`, which will remove the hold on the PR.
```

**Pull request merged**